.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -687,6 +687,14 @@ private DateTimeResolutionResult ParseOneWordPeriod(string text, DateObject refe
                     ret.Mod = Constants.MID_MOD;
                 }
 
+                // Handle 'eoy(end of year)', the behavior of 'eoy' should be the same as 'end of year'
+                if (match.Value.Equals("eoy"))
```

Comment:
Language-specific terms should never be hardcoded in base classes. Please add this to YAML and reference it through the appropriate config.